jupyterlab_gitplus/__init__.py

```diff
@@ -42,9 +45,24 @@ def _load_jupyter_server_extension(nb_server_app):
     host_pattern = '.*$'
 
     gh_token = nb_server_app.config.get('GitPlus', {}).get('github_token', '')
+    reviewnb_endpoint =  nb_server_app.config.get('GitPlus', {}).get(
+        'reviewnb_endpoint',
+        'https://app.reviewnb.com/'
+    )
+    try:
+        x = urlparse(reviewnb_endpoint)
+        if x.path not in ["", "/"]:
+            raise ValueError(f"reviewnb_endpoint URL must not have a path component. Probably you should change it to https://{x.netloc}/")
+        if not reviewnb_endpoint.endswith("/"):
+            reviewnb_endpoint += "/"
```

Comment:
It'd be worth adding a logger.info statement indicating the `reviewnb_endpoint` value set in the current set up.

Reply:
Done